src/Doctrine/EntityDetails.php

```diff
@@ -77,11 +77,18 @@ public function getFormFields(): array
             }
             $fieldsWithTypes[$fieldName] = [
                 'type' => EntityType::class,
-                'options_code' => sprintf('\'class\' => %s::class,', $relation['targetEntity']).\PHP_EOL.'\'choice_label\' => \'id\',',
+                'options_code' => <<<EOF
```

Comment:
This doesn't feel right. The bundled `php-cs-fixer` is run after generating templates and _should_  clean up any indentation issues. If it is not doing that, we need to figure out why. Having "indentation corrections" outside of a template file will be error prone.

Reply:
One way to tell if the indentation is related to something `php-cs-fixer` is _not_ doing, override the bundled fixer w/ a local install. Then run the command again and see if there's any change. 
https://github.com/symfony/maker-bundle/blob/8182644ba29b9545e1186edbe5a339f26afe69a3/src/Resources/config/services.xml#L44

The same would hold true for the `php-cs-fixer` config:
https://github.com/symfony/maker-bundle/blob/8182644ba29b9545e1186edbe5a339f26afe69a3/src/Resources/config/services.xml#L45

_If_ the results are same as with the bundled php-cs-fixer, it would suggest that php-cs-fixer is not running in this case for some reason...

Reply:
rule added